src/components/EventCard/EventCard.js

```diff
@@ -126,6 +132,14 @@ function EventCard({
 				</Grid>
 			</CardContent>
 			<CardActions className={classes.buttonArea}>
+				{ zoomLink ?
+					<Link href={isWithin12Hours ? zoomLink : null} underline="none">
+						<Button variant="outlined" size="small" color="secondary" disabled={!isWithin12Hours}>
```

Comment:
Having Button in Link is apparently bad for accessibility. Also, Gatsby's Link component [should not](https://www.gatsbyjs.com/docs/linking-between-pages/) be used with links to external websites, like Zoom links. Altogether, this calls for:

```js
				<Button variant="outlined" size="small" color="secondary" disabled={!isWithin12Hours}
					component='a' href={zoomLink} target='_blank' rel='noreferrer noopener'>
					Join
				</Button>
```

(Also, I think there is no need to set href to null if disabled is already set to true, now that the link is the same element as the Button.)

Reply:
We were using Material-UI's link already, which is fine for external links, but I can make this change 👍